ios/FluentUI/People Picker/AvatarView.swift

```diff
@@ -23,7 +23,7 @@ public enum AvatarSize: Int, CaseIterable {
     public var font: UIFont {
         switch self {
         case .extraSmall:
-            return UIFont.systemFont(ofSize: 9)
+            return UIFont.systemFont(ofSize: 8)
```

Comment:
but it should be 9 for regular case with out borders according to design?

Reply:
Ok, should the bordered case remain as a smaller font or should the internal border width change instead? The internal border is currently matching the design right now which I think cut off the extra small font

Reply:
that's what the figma redline shows. double check with Victor and Jackson who are the designers of this control.